website/docs/r/automanage_configuration_version.html.markdown

```diff
@@ -0,0 +1,183 @@
+---
+subcategory: "Automanage"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_automanage_configuration_version"
+description: |-
+  Manages an Automanage Configuration Profile Version.
+---
+
+# azurerm_automanage_configuration_version
+
+Manages an Automanage Configuration Profile Version. Any changes to the configuration profile could be supplied as properties in this resource.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-automanage"
+  location = "West Europe"
+}
+
+resource "azurerm_automanage_configuration" "example" {
+  name                = "example-acmp"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_automanage_configuration_version" "example" {
+  name                       = "version1"
+  resource_group_name        = azurerm_resource_group.example.name
+  location                   = azurerm_resource_group.example.location
+  configuration_profile_name = azurerm_automanage_configuration.example.name
+  # change the configuration profile to enable boot diagnostics
+  boot_diagnostics_enabled = true
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Automanage Configuration. Changing this forces a new Automanage Configuration to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Automanage Configuration should exist. Changing this forces a new Automanage Configuration to be created.
+
+* `location` - (Required) The Azure Region where the Automanage Configuration should exist. Changing this forces a new Automanage Configuration to be created.
+
+* `configuration_profile_name` - (Required) The name of the Automanage Configuration Profile where this version should exist. Changing this forces a new Automanage Configuration to be created.
+
+* `antimalware` - (Optional) A `antimalware` block as defined below.
```

Comment:
I intentially keep the same as https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/automanage_configuration#antimalware to avoid breaking change.